modules/webstat/lib/webstat_engine.py

```diff
@@ -1886,15 +1886,18 @@ def create_custom_summary_graph(data, path, title):
     """
     # If no input, we don't bother about anything
     if len(data) == 0:
-        return
+        return False
     os.environ['HOME'] = CFG_TMPDIR
 
     try:
         import matplotlib
         matplotlib.use('Agg')
         import matplotlib.pyplot as plt
-    except ImportError:
-        return
+    except:
+        from invenio.errorlib import register_exception
+        register_exception()
+
```

Comment:
It is OK to raise an exception, but let us keep `return` here, so that the page won't lead to Internal Server Error on installations that do not have matplotlib.  This was my thinking behind https://github.com/inveniosoftware/invenio/pull/2375#discussion_r22529417

If OK for you, I'll amend this when merging the PR.